### PR TITLE
translate to english

### PR DESCRIPTION
**Pull request by tsntsumi**

### CODE REVIEW

File: contents/basepages/aboutus/index.en.mdx [hunk 1]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+I was very particular about it and spent a lot of time making it, but I was worried about why no one was interested.
+I was selfish and even felt resentment, thinking that if people could hold it in their hands, look at it, and use it, they would understand how good it is.
+
+But I can't afford to open a store.
+I had no choice but to sell it online.
+Now you can't even pick it up.
+
+### Try to change your mind
+- "A change in perspective is worth 80 IQ points." -- Alan Key (A father of personal computer.)<br/>
+  "A change of perspective is worth 80 points of IQ." -- Alan Kay, father of the personal computer
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:57]: It's the same as the previous line, so please delete this line. Please also delete the <br/> in the previous line.

File: contents/basepages/aboutus/index.en.mdx [hunk 2]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+So I started studying online marketing.
+I am studying business administration and software engineering at university.
+It's because I've been doing internet related work for a long time.
+
+Also, rather than trying various superficial methods, I thought that exploring fundamental and holistic methods would provide me with long-lasting knowledge.
+If you use a superficial method, when the situation changes, you have to study from scratch again...
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:64]: Please make the subject "me" instead of "you".

File: contents/basepages/aboutus/index.en.mdx [hunk 3]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+date: 2023-01-19
+author: TSUTSUMI Kikuo
+slug: /aboutme
+images: 
+  - kikuo-profile-1-circle.png
+  - AliceNail-Testimonial.jpg
+  - kikuo-suffering.jpg
+showTOC: false
+---
+## self-introduction
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:19]: Please capitalized the first letter.

File: contents/basepages/aboutus/index.en.mdx [hunk 4]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+       image={0} bgColor="transparent"
+       style={{borderColor: "transparent"}} />
+
+Google Maps Acquisition Expert for Small Businesses
+- Learned internet marketing from sluggish sales of side business
+- Previously worked as an internet and web-related programmer
+- Majored in business administration, software engineering and artificial intelligence at university
+- Currently, we are spreading the "Google Maps method of attracting customers" that can attract customers for zero yen to small shop owners.
+
+### the beginning
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:31]: Please capitalized the first letter.

File: contents/basepages/aboutus/index.en.mdx [hunk 5]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+  "A change of perspective is worth 80 points of IQ." -- Alan Kay, father of the personal computer
+
+So I started studying online marketing.
+I am studying business administration and software engineering at university.
+It's because I've been doing internet related work for a long time.
+
+Also, rather than trying various superficial methods, I thought that exploring fundamental and holistic methods would provide me with long-lasting knowledge.
+If you use a superficial method, when the situation changes, you have to study from scratch again...
+
+### Meet the right way for small shops
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:66]: Please change "shop" to "business"

File: contents/basepages/aboutus/index.en.mdx [hunk 6]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+Then, many small shops and companies cannot sell because
+I realized that it was because I imitated TV commercials and magazine advertisements for large companies.
+In other words, there was a method of advertising, publicity, and sales that was optimal for the size of the store and the size of the trade area.
+
+It's called DRM (Direct Response Marketing).
+Moreover, it is similar to the way of thinking and way of thinking that was done in traditional Japanese business.
+
+Do you know Drucker, who is famous for "Moshi Dora"?
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:75]: Please change it to "Drucker, who is famous for their book called "Management""

File: contents/basepages/aboutus/index.en.mdx [hunk 7]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+It is said that Echigoya in the Edo period started the world's oldest marketing.
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:76]: Please change to “He said “

File: contents/basepages/aboutus/index.en.mdx [hunk 8]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+So, if you are Japanese, the content of my Google Maps attracting customers using DRM and DRM is
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:78]: Please delete "and DRM" in "DRM and DRM" in this line.

File: contents/basepages/aboutus/index.en.mdx [hunk 9]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+It's not easy for anyone to use.
+
+### Meet Google Maps, easy to use on your smartphone
+
+However, it is ideal for small local shops,
+Anyone can use it easily with a smartphone,
+And I found out about free Google business profiles.
+Google business profile
+It is a tool to manage the information of the shop placed on Google Maps.
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:92]: In MDX format files, multiple lines between empty lines and empty lines become one sentence. Please delete the line break when translating.

File: contents/basepages/aboutus/index.en.mdx [hunk 10]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+Instead, Google officially used "Local SEO"
+(Even if there is a search engine,
+There is no such thing as a map engine. )
+
+Local SEO is not local SEO.
+It's a local search engine optimization.
+Again, this is confusing, but the English word "local" doesn't seem to mean rural or country.
+Ultimately, it is a pair of words for global (whole), and it seems that regional and local are more appropriate.
+
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:109]: It's a sentence for Japanese people, so please delete it.

File: contents/basepages/aboutus/index.en.mdx [hunk 11]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+Then I started researching local SEO and started attracting customers on Google Maps.
+
+
+  Because I want to spread the way to let you know
+* If you make a mistake in notifying, not only will customers not come,
+  because i lose money
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:115]: Please be careful of line breaks and translate them again.

File: contents/basepages/aboutus/index.en.mdx [hunk 12]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+  ## People who have practiced Google Maps attracting customers...
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:116]: Please delete the blank at the beginning.

File: contents/basepages/aboutus/index.en.mdx [hunk 13]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+#### Before starting to attract customers to Google Maps
+
+<ImageBox alt="Testimonial from Nail Salon" float="right"
+       image={1} bgColor="transparent"
+       style={{borderWidth: "1px"}} />
+
+* No new acquisitions from 2 years ago
+* The number of existing customers has decreased sharply due to the corona crisis
+* I often feel depressed, and with my old mother in my hometown
+  I was beginning to think about living a simple and leisurely life.
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:128]: It's my client's story, so please make the subject "my client" or "her".

File: contents/basepages/aboutus/index.en.mdx [hunk 14]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+#### When you start attracting customers to Google Maps
+
+* Newly acquired for the first time in two years in just one week
+* Constant new acquisition just by uploading photos
+* Now he even wants to open a nail salon in his hometown.
+  I became positive and cheered up
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:135]: Please ignore the line break and translate the line.

File: contents/basepages/aboutus/index.en.mdx [hunk 15]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+### Single Grocery Store<br/> For A.K.
+
+#### Before starting to attract customers to Google Maps
+
+* The number of visitors has been decreasing since before the corona disaster
+* At night, I went to pubs and snack bars in the downtown area and sold snacks etc.
+
+#### When you start attracting customers to Google Maps
+
+*Inquiries from Google Maps increased
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:146]: Please add a blank space behind the "*"

File: contents/basepages/aboutus/index.en.mdx [hunk 16]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+* The number of inquiries has increased too much, and it has become a hindrance to work
+* Finally, I was able to attract customers to the point where I had to stop updating.
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:148]: Please change the subject to "her"

File: contents/basepages/aboutus/index.en.mdx [hunk 17]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+## Why is Tsutsumi doing what he is doing now?
+
+* She wants to help people who are suffering from attracting customers and selling like herself in the past.
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:152]: Please change the subject to "I"

File: contents/basepages/aboutus/index.en.mdx [hunk 18]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+* Even if there are wonderful products, services, and shops,
+  You can't attract customers if you don't let them know,
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:154]: Please ignore the line break and translate it again.

File: contents/basepages/aboutus/index.en.mdx [hunk 19]
```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
...
+
+## For those of you who are interested in the "Google Maps method of attracting customers" that allows you to aim for the number one store in your area just by taking a photo with your smartphone...
+
+Individual diagnosis that is usually offered at 5,000 yen for 60 minutes,
+Limited to those who apply from our website
+You can apply for a special price of 0 yen.
```
Comment (tsntsumi) [inline contents/basepages/aboutus/index.en.mdx:160]: Please ignore the line break and translate it again.

**Review by tsntsumi — COMMENTED** [review]
I wrote a comment where I wanted you to change it.

Please correct it.

File: contents/basepages/aboutus/index.en.mdx (hunk 2 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:64]: done

**Review by theahabla — COMMENTED** [review]
done

File: contents/basepages/aboutus/index.en.mdx (hunk 6 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:75]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 3 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:19]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 4 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:31]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 1 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:57]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 7 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:76]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 8 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:78]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 9 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:92]: done

File: contents/basepages/aboutus/index.en.mdx (hunk 10 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:109]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 11 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:115]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 12 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:116]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 13 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:128]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 14 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:135]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 15 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:146]: WHERE

File: contents/basepages/aboutus/index.en.mdx (hunk 5 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:66]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 16 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:148]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 18 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:154]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 15 above)
Comment (tsntsumi) [reply contents/basepages/aboutus/index.en.mdx:146]: > WHERE

"*Inquiries" to "* Inquiries"

File: contents/basepages/aboutus/index.en.mdx (hunk 17 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:152]: DONE

File: contents/basepages/aboutus/index.en.mdx (hunk 19 above)
Comment (theahabla) [reply contents/basepages/aboutus/index.en.mdx:160]: DONE

**Pull request merged**